sotodlib/coords/demod.py

```diff
@@ -180,4 +180,22 @@ def from_map(tod, signal_map, cuts=None, flip_gamma=True, wrap=False, modulated=
         if wrap:
             tod.wrap('signal', signal_sim, [(0, 'dets'), (1, 'samps')])
         return signal_sim
-    
\ No newline at end of file
+
+
+def rotate_demodQU(tod, zero_gamma=True):
```

Comment:
Can we change "zero_gamma" to "update_focal_plane"?  Then in the description say "whether to set focal_plane.gamma angles to zero, consistent with new coordinate reference."